src/main/java/com/hp/gaia/mgs/dto/BaseEvent.java

```diff
@@ -1,77 +0,0 @@
-package com.hp.gaia.mgs.dto;
-
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Map;
-
-/**
- * Created by belozovs on 7/15/2015.
- *
- * Base class for all kinds of events
- *
- */
-@JsonDeserialize(using = MainEventDeserializer.class)
```

Comment:
According to this benchmark, it's better to use GSON for small JSON files and Jackson for large files: http://blog.takipi.com/the-ultimate-json-library-json-simple-vs-gson-vs-jackson-vs-json/ 

I do not know which one is better for us, if we will use "bulks" it's probably going to be Jackson